components/dada-execute/src/step/apply_op.rs

```diff
@@ -67,31 +69,35 @@ impl Stepper<'_> {
             (&ObjectData::Int(lhs), &ObjectData::Int(rhs)) => match op {
                 Op::EqualEqual => Ok(self.machine.our_value(lhs == rhs)),
                 Op::Plus => match lhs.checked_add(rhs) {
-                    Some(value) => Ok(self.machine.our_value(value)),
+                    Some(value) => Ok(self.machine.our_value(ObjectData::Int(value))),
                     None => overflow_error(),
```

Comment:
I renamed `Int` to `SignedInt` and `UnsuffixedInt` to `Int`. I also changed the way two unsuffixed integers behave, now `5 - 3` results in an unsuffixed integer.